common/djangoapps/student/views.py

```diff
@@ -124,6 +124,7 @@
 
 # Note that this lives in openedx, so this dependency should be refactored.
 from openedx.core.djangoapps.credentials.utils import get_user_program_credentials
+from openedx.core.djangoapps.credit.email_utils import get_credit_provider_display_names
```

Comment:
This method was written by @awais786 in #11759.

@awais786 Can you move this method to any better place. ?